ToDoYandex/Views/ContentView.swift

```diff
@@ -0,0 +1,129 @@
+import SwiftUI
+
+struct ContentView: View {
+    @ObservedObject var contentViewModel: ContentViewModel
+    @State var isOpened = false
+    @State var itemId = ""
+    @State var showAllItems = true
+
+    private var doneItemsCount: Int {
+        contentViewModel.items.filter { $0.isDone }.count
+    }
+
+    init(contentViewModel: ContentViewModel) {
+        self.contentViewModel = contentViewModel
+    }
+
+    var body: some View {
+        NavigationStack {
+            ZStack {
+                VStack {
+                    HStack {
+                        Text("Выполнено - \(doneItemsCount)")
+                        Spacer()
+                        Button(action: {
+                            showAllItems.toggle()
+                        }) {
+                            Text(showAllItems ? "Скрыть" : "Показать")
+                                .foregroundColor(.blue)
+                                .padding(.trailing)
+                                .fontWeight(.semibold)
+                        }
+                    }
+                    .font(.system(size: 16))
+                    .padding(.horizontal)
+                    todoList
+                }
+                VStack {
+                    Spacer()
+                    addButton
+                }
+
+                .navigationBarTitle("Мои дела")
+                .sheet(isPresented: $isOpened) {
+                    TaskDetailsView(
+                        taskDetailsViewModel: TaskDetailsViewModel(
+                            fileCache: contentViewModel.fileCache, id: itemId
+                        )
+                    )
+                }
```

Comment:
[sheet(item:onDismiss:content:)](https://developer.apple.com/documentation/swiftui/view/sheet(item:ondismiss:content:))
Presents a sheet using the given item as a data source for the sheet’s content.